converters/fromZigbee.js

```diff
@@ -4370,6 +4370,26 @@ const converters = {
             return result;
         },
     },
+    aqara_switchtype: {
+        cluster: 'aqaraOpple',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            if (msg.data.hasOwnProperty(0x000A)) {
+                const lookup = {1: 'toggle', 2: 'momentary'};
+                return {switchtype: lookup[msg.data[0x000A]]};
+            }
+        },
+    },
+    aqara_power_outage_memory: {
```

Comment:
Please rename this to `xiaomi_switch_power_outage_memory` and add this to all devices using `tz.xiaomi_switch_power_outage_memory`